bucket/mambaforge.json

```diff
@@ -0,0 +1,57 @@
+{
+    "version": "4.12.0-0",
+    "description": "A conda-forge distribution",
+    "homepage": "https://mirrors.tuna.tsinghua.edu.cn/github-release/conda-forge/miniforge/",
+    "license": "BSD-3-Clause",
+    "notes": [
+        "From 4.6.0, conda has built the support for Cmd, Powershell or other shells.",
+        "Use \"conda init powershell\" or \"conda init __your_favorite_shell__\""
+    ],
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/conda-forge/miniforge/releases/download/4.12.0-0/Mambaforge-4.12.0-0-Windows-x86_64.exe",
+            "hash": "237f52e3a68eb3278d3c837763267840f23444e1b3762c00b1b8f01289d649c9"
+        }
+    },
+    "installer": {
+        "args": [
+            "/S",
+            "/InstallationType=JustMe",
+            "/RegisterPython=0",
+            "/AddToPath=0",
+            "/NoRegistry=1",
+            "/D=$dir"
+        ]
+    },
+    "uninstaller": {
+        "file": "Uninstall-Mambaforge.exe",
+        "args": "/S"
+    },
+    "bin": [
+        "python.exe",
+        "pythonw.exe",
+        [
+            "python.exe",
+            "python3"
+        ]
+    ],
+    "env_add_path": [
+        "scripts",
+        "Library\\bin"
+    ],
+    "persist": "envs",
+    "checkver": {
+        "github": "https://github.com/conda-forge/miniforge"
```

Comment:
```suggestion
        "github": "https://github.com/conda-forge/miniforge",
        "regex": "tag/([\\d.-]+)"
```

Reply:
`regex` is needed because Scoop matches `tag/[\\d.]+` by default. The version 4.12.0**-0** cannot be correctly matched.